package.json

```diff
@@ -26,7 +26,12 @@
     "ganache-cli": "^6.0.3",
     "js-sha3": "^0.7.0",
     "keythereum": "^1.0.2",
+<<<<<<< HEAD
```

Comment:
It seems like you committed an unresolved conflict.

It's a good idea to always test a PR thoroughly and look at the list of "files changed" before sending submitting it... If I had merged this without testing, it would have broken the project.